test/cql-pytest/test_guardrail_replication_strategy.py

```diff
@@ -20,20 +20,26 @@ def create_ks_and_assert_warnings_and_errors(cql, ks_opts, warnings_count=0, fai
         response_future = cql.execute_async("CREATE KEYSPACE " + keyspace + " " + ks_opts)
         response_future.result()
         cql.execute("DROP KEYSPACE " + keyspace)
-        if warnings_count:
-            assert response_future.warnings is not None and len(response_future.warnings) == warnings_count
-        else:
-            assert response_future.warnings is None
+
+        if warnings_count is not None:
```

Comment:
That if-ology, although correct, may prove a bit hard to follow for our future readers, maybe some extra explanatory comment?